packages/flutter_tools/test/asset_bundle_package_test.dart

```diff
@@ -48,13 +51,13 @@ $assetsSection
   }
 
   void establishFlutterRoot() {
-    // Setting flutterRoot here so that it picks up the MemoryFileSystem's
-    // path separator.
     Cache.flutterRoot = getFlutterRoot();
   }
 
   void writePackagesFile(String packages) {
-    fs.file('.packages')
+    final Uri uri = new Uri.file('.packages', windows: platform.isWindows);
```

Comment:
This shouldn't be necessary since file URIs only contain the `file:///` if they're absolute.  In this case, it'll just end up being `.packages`